demisto_sdk/commands/common/errors.py

```diff
@@ -1404,6 +1404,10 @@
         "code": "CR101",
         "related_field": "",
     },
+    "correlation_rules_execution_mode_error": {
```

Comment:
```suggestion
    "correlation_rules_execution_mode_empty: {
```
gives a tiny bit more info, the word `error` is redundant